frontend/src/metabase/common/components/EntityPicker/components/ItemList/ItemList.tsx

```diff
@@ -78,12 +78,22 @@ export const ItemList = <
         const isSelected = isSelectedItem(item, selectedItem);
         const icon = getEntityPickerIcon(item, isSelected && isCurrentLevel);
 
+        const folderChevronColor = isSelected
+          ? "var(--mb-color-text-white)"
+          : "var(--mb-color-text-primary)";
```

Comment:
Allows the right chevron color of unselected items to be themed:

<img src="https://github.com/user-attachments/assets/a55dceab-5f36-4fcf-8ef1-954f6311e009" width="300">

Reply:
Nice catch! Fixed in https://github.com/metabase/metabase/pull/50449/commits/6e4599e4c607de8d1eb57f3338966eed629880dc